KUADRANT.md

```diff
@@ -301,7 +304,7 @@ permission:
     policies-csv-file: ../../rbac-policy.csv
 ```
 
-The `userEntityRef` determines which user you're logged in as (changeable via `switch-user.sh`).
+The `userEntityRef` determines which user you're logged in as (changeable via yarn user commands).
```

Comment:
I think you can delete this line. The new yarn command updates group ownership instead of user entity ref. 

The new ` userEntityRef` is `user:default/guest`.